app/parse.py

```diff
@@ -8,5 +16,39 @@
     duration: str
 
 
+def get_course_info_soup(course_soup: BeautifulSoup) -> BeautifulSoup:
+    course_info_page = course_soup.select_one("a")["href"]
+    course_url = urljoin(BASE_URL, course_info_page)
+
+    course_page = requests.get(course_url).content
+
+    course_info_soup = BeautifulSoup(course_page, "html.parser")
+
+    return course_info_soup
+
+
+def parse_course_name(course_soup: BeautifulSoup) -> Course:
+    return Course(
+        name=course_soup.select_one("a")["title"],
+        short_description=get_course_info_soup(course_soup).find(
+            class_="typography_landingTextMain__Rc8BD"
+                   " SalarySection_aboutProfession__1VFHK"
+        ).text,
+        duration=get_course_info_soup(course_soup).find_all(
+            class_="ComparisonTable_cell__RNsyU ComparisonTable_highl"
+                   "ightCell__IpyWZ typography_newLandingH6__Av9tT"
+        )[12].text
```

Comment:
The `get_course_info_soup(course_soup).find_all()` call assumes that the list of elements contains at least 13 items, as it accesses the 13th element with `[12]`. If there are fewer elements, this will raise an `IndexError`. Consider adding a check to ensure the list has enough elements before accessing the 13th item.